src/features/MovieList/index.js

```diff
@@ -1,28 +1,42 @@
+import { useDispatch, useSelector } from "react-redux";
+import { useEffect } from "react";
+
 import MovieTiles from "../Tiles/MovieTiles";
 import { useAPIData } from "../../useAPIData";
 import { Loader } from "../../common/Loader";
 import Error from "../../common/Error";
 import { NoResults } from "../../common/NoResults";
 import { Wrapper } from "../MovieDetails/styled";
 import { Pagination } from "../MovieList/Pagination";
+import { useQueryParameter } from "../../useQueryParameter";
+import searchQueryParamName from "../../searchQueryParamName";
+import { selectPopularMoviesStatus, fetchPopularMoviesLoading } from "./popularMoviesSlice";
 
 const MovieList = () => {
+    const dispatch = useDispatch();
+    const status = useSelector(selectPopularMoviesStatus);
+
+    const page = useQueryParameter(searchQueryParamName) || "1";
+
+    useEffect(() => {
+        dispatch(fetchPopularMoviesLoading(page));
+    }, [dispatch, page]);
+
     const APIData = useAPIData();
 
     const MovieListContent = () => {
-        switch (APIData.status) {
+        switch (status) {
             case "loading":
                 return < Loader />;
             case "success":
                 return (
                     <>
                         <Wrapper>
                             <MovieTiles title="Popular movies"
-                                popularMoviesData={APIData.popularMoviesData.data}
                                 movieData={APIData.movieData.data}
```

Comment:
if you want to use redux you don't need props here.

Reply:
I mean you probably still need `title="Popular movies"`

Reply:
yes, but after adding next api data - we need genres. its to be changed in next step

Reply:
👍